conf/firmwares/subsystems/rotorcraft/ins_hff.makefile

```diff
@@ -7,7 +7,7 @@ $(TARGET).srcs += $(SRC_SUBSYSTEMS)/ins.c
 $(TARGET).srcs += $(SRC_SUBSYSTEMS)/ins/ins_int.c
 
 # vertical filter float version
-$(TARGET).srcs += $(SRC_SUBSYSTEMS)/ins/vf_float.c
+#$(TARGET).srcs += $(SRC_SUBSYSTEMS)/ins/vf_float.c
```

Comment:
This is breaking things for everyone else. At least make a configuration option to disable it, but don't remove it.